udata/core/dataset/constants.py

```diff
@@ -4,42 +4,62 @@
 
 #: Udata frequencies with their labels
 #:
-#: See: http://dublincore.org/groups/collections/frequency/
+#: See: https://publications.europa.eu/en/web/eu-vocabularies/at-dataset/-/resource/dataset/frequency
 UPDATE_FREQUENCIES = OrderedDict(
-    [  # Dublin core equivalent
-        ("unknown", _("Unknown")),  # N/A
-        ("punctual", _("Punctual")),  # N/A
-        ("continuous", _("Real time")),  # freq:continuous
-        ("hourly", _("Hourly")),  # N/A
-        ("fourTimesADay", _("Four times a day")),  # N/A
-        ("threeTimesADay", _("Three times a day")),  # N/A
-        ("semidaily", _("Semidaily")),  # N/A
-        ("daily", _("Daily")),  # freq:daily
-        ("fourTimesAWeek", _("Four times a week")),  # N/A
-        ("threeTimesAWeek", _("Three times a week")),  # freq:threeTimesAWeek
-        ("semiweekly", _("Semiweekly")),  # freq:semiweekly
-        ("weekly", _("Weekly")),  # freq:weekly
-        ("biweekly", _("Biweekly")),  # freq:bimonthly
-        ("threeTimesAMonth", _("Three times a month")),  # freq:threeTimesAMonth
-        ("semimonthly", _("Semimonthly")),  # freq:semimonthly
-        ("monthly", _("Monthly")),  # freq:monthly
-        ("bimonthly", _("Bimonthly")),  # freq:bimonthly
-        ("quarterly", _("Quarterly")),  # freq:quarterly
-        ("threeTimesAYear", _("Three times a year")),  # freq:threeTimesAYear
-        ("semiannual", _("Biannual")),  # freq:semiannual
-        ("annual", _("Annual")),  # freq:annual
-        ("biennial", _("Biennial")),  # freq:biennial
-        ("triennial", _("Triennial")),  # freq:triennial
-        ("quinquennial", _("Quinquennial")),  # N/A
-        ("irregular", _("Irregular")),  # freq:irregular
+    [
+        # FIXME: Add ALL EU terms (from 30yrs to 1mn)?
+        # FIXME: Uppercase to match EU vocab?
```

Comment:
Need to decide on this. Do we want to stick with EU's uppercasing (enum-like)? Or lowercase to stay consistent with udata API output?

We could also go with an enum instead of a dict here => uppercase for the enum values, but output lowercase in the API response.